test/tools/yulInterpreter/EwasmBuiltinInterpreter.cpp

```diff
@@ -133,7 +130,18 @@ u256 EwasmBuiltinInterpreter::evalBuiltin(YulString _fun, vector<u256> const& _a
 		accessMemory(arg[0], 4);
 		return readMemoryHalfWord(arg[0]);
 	}
-
+	else if (_fun == "i32.clz"_yulstring)
+	{
+		if (arg[0] == 0)
+			return 32;
+		return clz(arg[0]);
```

Comment:
This looks inconsistent to me - shouldn't it be `clz(arg[0]) - 32`?

Reply:
True! I'd rather change clz though, but can do it here.

Reply:
Fixed.